### PR TITLE
chore(macros/JsSidebar): add "Trailing commas" page

### PR DESCRIPTION
**Pull request by Josh-Cena**
## Summary

This page is not well linked to. Placing it in the sidebar will increase visibility.

### Solution

---

### After

---

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
👍 From the content point of view.

File: kumascript/macros/JsSidebar.ejs [hunk 1]
```diff
@@ -173,6 +175,7 @@ var text = mdn.localStringMap({
     'Iteration_protocols': 'Protocoles d\'itération',
     'Transitioning_to_strict_mode': 'Passer au mode strict',
     'Template_strings': 'Gabarits de chaînes de caractères',
+    'Trailing_commas': 'Trailing commas',
```
Comment (caugner) [inline kumascript/macros/JsSidebar.ejs:178]:
```suggestion
    'Trailing_commas': 'Virgules finales',
```

File: kumascript/macros/JsSidebar.ejs [hunk 2]
```diff
@@ -118,6 +119,7 @@ var text = mdn.localStringMap({
     'Iteration_protocols': 'Протоколы перебора',
     'Transitioning_to_strict_mode': 'Переход в строгий режим',
     'Template_strings': 'Шаблонные строки',
+    'Trailing_commas': 'Trailing commas',
```
Comment (caugner) [inline kumascript/macros/JsSidebar.ejs:122]:
```suggestion
    'Trailing_commas': 'Висящие запятые',
```

File: kumascript/macros/JsSidebar.ejs [hunk 3]
```diff
@@ -283,6 +287,7 @@ var text = mdn.localStringMap({
     'Iteration_protocols': '反復処理プロトコル',
     'Transitioning_to_strict_mode': 'strict モードへの移行',
     'Template_strings': 'テンプレート文字列',
+    'Trailing_commas': 'Trailing commas',
```
Comment (caugner) [inline kumascript/macros/JsSidebar.ejs:290]:
```suggestion
    'Trailing_commas': '末尾のカンマ',
```

**Review by caugner — CHANGES_REQUESTED** [review]
LGTM, but the page is already localized in fr/ja/ru/zh-CN, so let's reuse those localized titles.

**Review by caugner — APPROVED** [review]

**Comment by caugner** [comment]
@Josh-Cena You might also want to add it to the parent page: [JavaScript reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference), maybe in the [Additional reference pages](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference#additional_reference_pages) section.

**Pull request merged**